test/uitest.c

```diff
@@ -59,9 +59,11 @@ static int test_old()
     char pass[16];
     int ok = 0;
 
-    if ((ui_method =
-         UI_UTIL_wrap_read_pem_callback(test_pem_password_cb, 0)) == NULL
-        || (ui = UI_new_method(ui_method)) == NULL)
+    ui_method = UI_UTIL_wrap_read_pem_callback(test_pem_password_cb, 0);
+    if (!TEST_ptr(ui_method))
+        goto err;
+    ui = UI_new_method(ui_method);
+    if (!TEST_ptr(ui))
```

Comment:
Do you want me to do that?  It makes me uneasy, since I am used to not putting side effects in macro's.  And since the file/line-number are reported, I am not sure how necessary it is.  But let me know.